README.md

```diff
@@ -108,11 +108,11 @@ If you want to point it at your own GCC or similar binaries, either edit the `et
 else make a new one with the name `LANG.local.properties`, substituting `LANG` as needed. `*.local.properties` files
 have the highest priority when loading properties.
 
-If you want to support multiple compilers and languages like [godbolt.org](https://godbolt.org), you can use the
-`bin/ce_install install compilers` command in the [infra](https://github.com/compiler-explorer/infra) project to install
-all or some of the compilers. Compilers installed in this way can be loaded through the configuration in
-`etc/config/*.amazon.properties`. If you need to deploy in a completely offline environment, you may need to remove some
-parts of the configuration that are pulled from `www.godbolt.ms@443`.
+If you want to support multiple compilers and languages like compiler-explorer.com](https://compiler-explorer.com), you
+can use the `bin/ce_install install compilers` command in the [infra](https://github.com/compiler-explorer/infra)
+project to install all or some of the compilers. Compilers installed in this way can be loaded through the configuration
+in `etc/config/*.amazon.properties`. If you need to deploy in a completely offline environment, you may need to remove
+some parts of the configuration that are pulled from `www.godbolt.ms@443`.
```

Comment:
(retry comment) - this is no longer true; the urls are
godbolt.org@443/gpu and godbolt.org@443/winprod

Reply:
I don't think the README is misleading as is, but fixing anyway.